CI: build failing because of gbq testing

see https://github.com/pandas-dev/pandas/pull/25554#issuecomment-477602786

added back the gbq dep and this is now failing.

Seems to be https://github.com/conda-forge/pandas-gbq-feedstock/issues/18, compare [failed job](https://travis-ci.org/pandas-dev/pandas/jobs/512494733).

```
    def _test_google_api_imports():
    
        try:
            import pydata_google_auth
        except ImportError as ex:
            raise ImportError(
>               "pandas-gbq requires pydata-google-auth: {0}".format(ex)
            )
E           ImportError: pandas-gbq requires pydata-google-auth: No module named 'pydata_google_auth'
```

Note that `pandas-gbq` wasn't tested anywhere in the CI between #24942 was merged and now (after #25554). Seems we're still getting the old recipe somehow, because this is failing on the unpinned build. Simplest solution is to add `pydata-google-auth` to conda deps for the time being.

--- a/ci/deps/travis-36-cov.yaml
+++ b/ci/deps/travis-36-cov.yaml
@@ -19,6 +19,9 @@ dependencies:
   - pandas-gbq
   - psycopg2
   - pyarrow=0.9.0
+  # add pydata-google-auth until for pandas-gbq, see
+  # https://github.com/conda-forge/pandas-gbq-feedstock/issues/18
+  - pydata-google-auth
   - pymysql
   - pytables
   - python-snappy

